admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -59,7 +59,7 @@ $(document).ready(function() {
   /* validate price fields , as Thomas de Nabord said */
   $(".money-type input[type='text']").change(function validate(event) {
     var inputValue = $(this).val();
-    var parsedValue = truncateDecimals(parseFloat(inputValue),6);
+    var parsedValue = (new Number(inputValue)).toFixed(6);
```

Comment:
Won't work, or will result to a bad round value after 6 decimals.

For instance, try with 49.999999999999